core/ingest/src/main/java/mil/nga/giat/geowave/core/ingest/s3/S3URLConnection.java

```diff
@@ -67,6 +67,8 @@ private ClientConfiguration buildClientConfig() {
 				PROP_S3_HANDLER_SIGNER_OVERRIDE,
 				null);
 
+		// HP Fortify "Portability Flaw" false positive
+		// This is purposefully kept dynamic so help solve portability problems.
 		final ClientConfiguration clientConfig = new ClientConfiguration()
```

Comment:
lets just delete toLowerCase on line 65 and change equals on line 73 to equalsIgnoreCase()